pontoon/base/static/css/check-box.css

```diff
@@ -4,33 +4,33 @@
   margin: 0;
   display: inline-block;
   text-align: left;
-}
 
-#main .check-list .check-box {
-  padding: 8px 0;
-}
+  .check-box {
+    padding: 8px 0;
+  }
```

Comment:
The syntax for CSS nesting requires an `&` nesting selector, unlike how it works with SCSS & less: https://kilianvalkhof.com/2021/css-html/css-nesting-specificity-and-you/

Reply:
That seems to be optional according to MDN:
https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_nesting/Using_CSS_nesting